Code/Tools/TestImpactFramework/Frontend/Console/Code/Source/TestImpactCommandLineOptions.cpp

```diff
@@ -111,6 +114,16 @@ namespace TestImpact
             return ParsePathOption(OptionKeys[SequenceReportKey], cmd);
         }
 
+        AZStd::optional<AZStd::vector<AZStd::string>> ParseExcludeTestsFile(const AZ::CommandLine& cmd)
```

Comment:
suggestion: Having an `optional` vector might be overkill here as we can simply check if the vector is `.empty()` to see if any strings were generated. That would basically be the 'empty optional' state and is likely more consistent with other APIs across the source code

Reply:
This has now been changed in the most recent push - I had already solved this but hadn't pushed my changes into the repo, apologies.

Reply:
Ah nice one, thanks for following-up 👍